packages/chat-component-bindings/src/messageThreadSelector.ts

```diff
@@ -28,11 +28,10 @@ import { ACSKnownMessageType } from './utils/constants';
 /* @conditional-compile-remove(data-loss-prevention) */
 import { DEFAULT_DATA_LOSS_PREVENTION_POLICY_URL } from './utils/constants';
 import { updateMessagesWithAttached } from './utils/updateMessagesWithAttached';
-
 /* @conditional-compile-remove(file-sharing) @conditional-compile-remove(teams-inline-images-and-file-sharing) */
-import { FileMetadata, ChatAttachmentType, InlineImageMetadata } from '@internal/react-components';
+import { FileMetadata, ChatAttachmentType as AttachmentType, InlineImageMetadata } from '@internal/react-components';
```

Comment:
Is this because of the naming collision?

Reply:
> Is this because of the naming collision?

@palatter  Yes. Now that we have the same type name from both our library and ChatSDK, we have to make the distinguish here.